configs/eslint-plugin-compass/rules/no-mongodb-link-without-utm-params.js

```diff
@@ -0,0 +1,63 @@
+const isMongodbURL = (str) => {
+  try {
+    const url = new URL(str);
+    return url.hostname.includes('mongodb');
+  } catch (error) {
+    return false;
+  }
+};
+
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description:
+        'Links pointing to *.mongodb.com must have utm_source=compass and utm_medium=product in query params',
+    },
+    fixable: 'code',
+    hasSuggestions: true,
+  },
+
+  create(context) {
+    const isTestFile = /\.(test|spec)\.(js|jsx|ts|tsx)$/.test(
+      context.getPhysicalFilename()
+    );
+    return {
+      Literal(node) {
```

Comment:
Would be nice also make it work with `TemplateLiteral` type

Reply:
This was pretty nice to look into 🤩 I added this as well in my recent commits